azurerm/resource_arm_dev_test_policy_test.go

```diff
@@ -0,0 +1,175 @@
+		labName := rs.Primary.Attributes["lab_name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+
+		conn := testAccProvider.Meta().(*ArmClient).devTestPoliciesClient
+		ctx := testAccProvider.Meta().(*ArmClient).StopContext
+
+		resp, err := conn.Get(ctx, resourceGroup, labName, policySetName, policyName, "")
+		if err != nil {
+			return fmt.Errorf("Bad: Get devTestPoliciesClient: %+v", err)
+		}
+
+		if resp.StatusCode == http.StatusNotFound {
+			return fmt.Errorf("Bad: DevTest Policy %q (Policy Set %q / Lab %q / Resource Group: %q) does not exist", policyName, policySetName, labName, resourceGroup)
+		}
+
+		return nil
+	}
+}
+
+func testCheckAzureRMDevTestPolicyDestroy(s *terraform.State) error {
+	conn := testAccProvider.Meta().(*ArmClient).devTestPoliciesClient
+	ctx := testAccProvider.Meta().(*ArmClient).StopContext
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "azurerm_dev_test_policy" {
+			continue
+		}
+
+		policyName := rs.Primary.Attributes["name"]
+		policySetName := rs.Primary.Attributes["policy_set_name"]
+		labName := rs.Primary.Attributes["lab_name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+
+		resp, err := conn.Get(ctx, resourceGroup, labName, policySetName, policyName, "")
+
+		if err != nil {
+			if resp.StatusCode == http.StatusNotFound {
+				return nil
+			}
+
+			return err
+		}
+
+		return fmt.Errorf("DevTest Policy still exists:\n%#v", resp)
+	}
+
+	return nil
+}
+
+func testAccAzureRMDevTestPolicy_basic(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_dev_test_lab" "test" {
+  name                = "acctestdtl%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+resource "azurerm_dev_test_policy" "test" {
+  name                = "LabVmCount"
+  policy_set_name     = "default"
+  lab_name            = "${azurerm_dev_test_lab.test.name}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  threshold           = "999"
+  evaluator_type      = "MaxValuePolicy"
+}
+`, rInt, location, rInt)
+}
+
+func testAccAzureRMDevTestPolicy_complete(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_dev_test_lab" "test" {
+  name                = "acctestdtl%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+resource "azurerm_dev_test_policy" "test" {
+  name                = "LabVmCount"
+  policy_set_name     = "default"
+  lab_name            = "${azurerm_dev_test_lab.test.name}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  threshold           = "999"
+  evaluator_type      = "MaxValuePolicy"
+
```

Comment:
it seems `fact_name` and `description` are omitted?

Reply:
`fact_name` only applies to certain configurations (which would require additional resources) - but we can add description here 👍